inc/classes/admin/script/loader.class.php

```diff
@@ -539,12 +539,21 @@ public static function get_gutenberg_compat_scripts() {
 			[
 				'id'       => 'tsf-gbc',
 				'type'     => 'js',
-				'deps'     => [ 'jquery', 'tsf', 'tsf-utils', 'wp-editor', 'wp-data', 'react' ],
+				'deps'     => [ 'jquery', 'tsf', 'tsf-utils', 'wp-editor', 'wp-data', 'react', 'wp-element', 'wp-components' ],
```

Comment:
Tangential note: I don’t think 'react' was needed in the deps. It’s certainly not now that 'wp-element' is in there.